src/SprykerEco/Zed/AfterPay/Persistence/AfterPayQueryContainerInterface.php

```diff
@@ -17,6 +17,9 @@
 interface AfterPayQueryContainerInterface extends QueryContainerInterface
 {
     /**
+     * Specification:
```

Comment:
* Add a specification to a methods.
Example:
* Returns `SpyPaymentAfterPayQuery` query for a given `idSalesOrder`.